src/containers/Banner.tsx

```diff
@@ -76,25 +77,38 @@ const StyledInput = styled.input`
 
 interface BannerProps {
   isEditBanner: boolean;
```

Comment:
If you add an option, you don't have to write in isEditBanner={false} props, bcs undefined === false, right?
```suggestion
  isEditBanner?: boolean;
```

Reply:
True. Fixed.